English translation completed (#124)
vivliostyle/vivliostyle.org: _posts/2022-07-28-vivliostyle-pub-now-exports-pdf-files.md

---
title: Vivliostyle Pub now exports PDF files
lang: en
image: /assets/posts/2022-07-28-vivliostyle-pub-now-exports-pdf -files/fig-1.png
author:
  - ogwata
---

<div style="float: right; margin: 0 0 1em 1em;"><img src="/assets/posts/2022-07-28-vivliostyle-pub-now-exports-pdf -files/fig-1.png" alt="Vivliostyle Pub now exports PDF files" style="width: 500px;" /></div>

Vivliostyle Pub released an alpha version in April. Since then, development has proceeded smoothly, and just recently it has become possible to export to PDF files. The user guide has been substantially revised accordingly.

> - [Vivliostyle Pub Alpha version User Guide](https://vivliostyle.github.io/docs-vivliostyle-pub/#/ja/) (Sorry, the English translation has not yet been translated)

Reviewer: 「the English translation has not yet been translated (英語訳はまだ翻訳されていない)」は、英語訳から別の言語に翻訳しようとしているみたいで不自然です。「the English translation is still under preparation (英語訳はまだ準備中です)」のほうがよいでしょう。

```suggestion
- [Vivliostyle Pub Alpha version User Guide](https://vivliostyle.github.io/docs-vivliostyle-pub/#/ja/) (Sorry, the English translation is still under preparation.)
```

- [You can apply any theme (style sheet) to your Markdown file.](https://vivliostyle.github.io/docs-vivliostyle-pub/#/ja/functions-of-the-actions-menu/theme)
- [You can use Google Fonts that can be used free of charge and various paid web font services.](https://vivliostyle.github.io/docs-vivliostyle-pub/#/ja/create-and-save-documents/how-to-specify-fonts#custom-theme%EF%BC%8Fgoogle%E3%83%95%E3%82%A9%E3%83%B3%E3%83%88%E3%81%AE%E4%BD%BF%E7%94%A8)
- [You can combine multiple Markdown files into one book.](https://vivliostyle.github.io/docs-vivliostyle-pub/#/ja/create-and-save-documents/document-customization.md#%E5%AF%BE%E8%B1%A1%E3%81%A8%E3%81%AA%E3%82%8B%E6%96%87%E6%9B%B8%E3%81%AE%E6%8C%87%E5%AE%9A)
- [You can edit with multiple users using the features of GitHub.](https://vivliostyle.github.io/docs-vivliostyle-pub/#/ja/multi-user-collaborative-editing/overview)
- [You can export PDF files.](https://vivliostyle.github.io/docs-vivliostyle-pub/#/ja/functions-of-the-actions-menu/export#pdf)

> However, due to the specification that the typesetting engines responsible for preview and PDF output are different, the fonts used in preview and those used in PDF output may not match, and the pages may differ from each other.

Reviewer: > due to the specification that the op environments  typesetting engines responsible for preview and PDF output are different, 

ここを読んで元の日本語の「プレビューとPDF出力を担当する組版エンジンが異なるという仕様により、」というのも表現を見直す必要があると思いました。

組版エンジンはどちらも同じVivliostyle.jsです。組版エンジンが異なるのではなくて、組版エンジンが動作する実行環境が異なるというのが正しいです。

日本語の方は「プレビューとPDF出力とで実行環境が異なるため、」に直して、英語は「Because the running environment differs between preview and PDF output,」に直すとよいと思います。

Author: > 組版エンジンはどちらも同じVivliostyle.jsです。組版エンジンが異なるのではなくて、組版エンジンが動作する実行環境が異なるというのが正しいです。

ご指摘感謝です。たしかに「組版エンジンが異なる」は修正の必要がありますね。
ただ、ご提案の「実行環境」という言葉は抽象的で、いったい何が異なるのかユーザーがイメージしづらいのが難点です。以下ではいかがでしょう？

和文
ただし、プレビューとPDF出力とでは組版エンジンの場所が異なるという仕様により、場合によってプレビューで使われるフォントとPDF出力で使われるフォントが一致せず、これにより互いにページが食い違ってしまうことがあり得ます。

英文
However, due to the different location of the typesetting engine between preview and PDF output, the fonts used in preview and PDF output may not match, and this may cause pages to differ from each other.

Reviewer: 👍
